nsv13/code/game/gamemodes/overmap/objectives/board_ship.dm

```diff
@@ -19,8 +19,8 @@
 	target_ship.name = ship_name
 	// give it a home
 	var/list/candidates = list()
-	for(var/datum/star_system/random/S in SSstar_system.systems)
-		if(!S.hidden)
+	for(var/datum/star_system/S in SSstar_system.neutral_zone_systems)
+		if(!S.hidden && !length(S.fleets) && (!S.get_info()?["Black hole"]) && (S != SSstar_system.system_by_id(SSovermap_mode.mode.starting_system)))
```

Comment:
```suggestion
		if(!S.hidden && !length(S.fleets) && (!S.get_info()?["Black hole"]) && (S != SSstar_system.system_by_id(SSovermap_mode.mode.starting_system)) && (S.faction == target_ship.faction || "unaligned" || "uncharted")
```
The madness cannot be stopped (we don't want the ship spawning *inside friendly claimed territory*) keep in mind it checks for system *faction*, meaning it will only spawn in unoccupied systems (or those occupied by the allied faction) to avoid being blown to bits by enemy ships. We can change this to *owner* if we want to make sure it can spawn in those situations, the fleet accompanying it might be able to defend it just fine. Just something to test I guess.